modules/installation-guide/partials/proc_using-configmap-to-configure-the-number-of-workspaces-that-a-user-can-run.adoc

```diff
@@ -13,22 +14,20 @@ Each new workspace creates at least one new mount point to the storage (PV), in
 --
 +
 For more information, see xref:configuring-storage-strategies.adoc[].
-* Determine the values of the following placeholders:
+* You have determined the value of the following placeholder:
 + 
 [cols="1,2"]
 |===
-| `_<product-namespace>_`
-| The namespace of the product. The default option for `{platforms-namespace}` is `{prod-namespace}`.
-
 | `_<number-of-workspaces>_`
-| The number of workspaces a user can run simultaneously. The default value is `1`. If the value is `-1`, an unlimited number of workspaces can run per user. If it is a precise positive value, that many workspaces can run per user.
+| The number of workspaces that a user can run simultaneously. If the value is `-1`, an unlimited number of workspaces can run per user. If the value is a positive integer, the users can run that many workspaces. The default value is `1`.
 |===
 
 .Procedure
-. Configure the number of workspaces a user can run simultaneously:
+
+. Configure the number of workspaces that a user can run simultaneously:
```

Comment:
by running the following command

Reply:
since this is a procedure and not general information, would "configure your number of workspaces" make sense? Or is this some sysadmin thing, where you determine what the users can and can't do (obviously clueless here :D)

Reply:
The procedure sentence could instead provide more details as to what action exactly (to type/save) or where exactly (in which file) the user is supposed to perform, so as not to repeat the same phrase from the heading and intro par.